blocks/layout-grid/editor.scss

```diff
@@ -209,3 +209,33 @@
 		}
 	}
 }
+
+/**
+ * Individual column alignment for the editor
+ */
+.wp-block-jetpack-layout-grid {
+	&.are-vertically-aligned-top [data-type="jetpack/layout-grid-column"].wp-block {
+		align-self: flex-start;
+	}
+
+	&.are-vertically-aligned-center [data-type="jetpack/layout-grid-column"].wp-block {
```

Comment:
Should we rename these though? I know the grammar may not be as pure, but to me `is-vertically-aligned-*` sounds better than `are-vertically-aligned-*`, even if it refers to the content inside the column rather than the column itself. This is what the columns block also uses: `is-vertically-aligned-bottom`

Reply:
Both names are taken from the columns block - `is-vertically-aligned` is for an individual column, and `are-vertically-aligned` is for the column block itself.

https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/columns/style.scss#L77

Reply:
Well then. Ignore me!